_posts/0000-01-02-junaydirfan.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
It's good to be change some things sometimes. Well, that's what I tell people anyway.